### ISSUE TITLE
Focus indicator on 'selected' test in Fast Pass has insufficient contrast against its background

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#868]
The following AI-windows mode(s) have this issue:
- High Contrast White mode
- High Contrast Black mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![15 1](https://user-images.githubusercontent.com/22068434/88356099-8f49a200-cd1b-11ea-9852-af0946f498ea.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by iamrafan** [comment]
Validated the fixes in Canary Release: v1.1.1388.01. All 6 modes are now compliant
